lib/plugins/output-filter/kubernetes-enrichment.js

```diff
@@ -186,7 +186,7 @@ function replaceDockerImageName (pod, data) {
       return
     }
     if (data.container.image.name === 'sha256') {
-      data.container.image.digest = 'sha256:' + data.container.image.tag
+      //data.container.image.digest = 'sha256:' + data.container.image.tag
```

Comment:
Since you don't know where the tag is coming from and this does not match what we see in the frontend, I suggest you first check what's coming from the container itself before removing the processing. 

```suggestion
      console.log(`Container name:  ${data.container.image.name}`)
      console.log(`Container tag: ${data.container.image.tag}`)
      data.container.image.digest = 'sha256:' + data.container.image.tag
```